src/main/java/com/fasterxml/jackson/databind/DeserializationFeature.java

```diff
@@ -406,10 +406,10 @@ public enum DeserializationFeature implements ConfigFeature
      * Feature that allows unknown Enum values to be parsed as null values. 
      * If disabled, unknown Enum values will throw exceptions.
      *<p>
-     * Note that in some cases this will basically ignore unknown Enum values;
-     * this is the keys for keys of {@link java.util.EnumMap} and values
-     * of {@link java.util.EnumSet} (because nulls are not accepted in these
-     * cases).
+     * Note that in some cases this will in effect ignore unknown Enum values,
+     * e.g. when the unknown values are used as keys of {@link java.util.EnumMap} 
+     * or values of {@link java.util.EnumSet}, given the current deserializer
```

Comment:
Referring to https://github.com/FasterXML/jackson-databind/blob/2.14/src/main/java/com/fasterxml/jackson/databind/deser/std/EnumMapDeserializer.java#L299-L301

Reply:
If I remember correctly, the problem is not deserializer implementation as much as the resulting `EnumMap` and `EnumSet` that cannot contain `null` values. Otherwise correct and more fluent description, thanks!